src/aequitas/preprocessing.py

```diff
@@ -70,14 +66,18 @@ def preprocess_input_df(df, required_cols=None):
         required_cols = ['score']
     try:
         check_required_cols(df, required_cols)
-    except ValueError:
-        logging.error('preprocessing.preprocess_input_df: input dataframe does not have all the required columns.')
+    except ValueError as e:
```

Comment:
Yes, certainly does seem to make more sense for the exception to be propagated. (I presume squashing it was a mistake from a previous clean-up.)

I am curious if there's value here in logging a special message at all. Presumably the user will get the traceback and error message from the uncaught exception.

If it's important that this go to the log (which it certainly might be), the logger has shortcuts for this, _e.g._:

```suggestion
    except ValueError:
        logging.exception('input dataframe does not have all the required columns')
        raise
```

The above will log the traceback and exception along with the friendly message, and then propagate the exception for the interpreter to print the traceback and exception as well.